src/components/PriceHistory.vue

```diff
@@ -49,6 +49,9 @@ export default {
         min: this.daysAgo === 0 ? 0 : start,
       };
 
+      const maxYTick = this.formatedData('Buy')
+        .reduce((prev, current) => ((prev.y > current.y) ? prev : current)).y * 1.1;
```

Comment:
Proposing a shorter way:
```suggestion
      const maxYTick = Math.max(...this.formatedData('Buy').map(({ y }) => y)) * 1.1;
```